scripts/fetch/fetch_utils.R

```diff
@@ -0,0 +1,62 @@
+# Fetch utility functions
+
+#' Convert a vector of counties into sf polygons
+#' 
+#' counties must be strings of "[state]:[county]". See ?maps::county.fips
+#' 
+county_to_sf <- function(counties, crs = 3086){
```

Comment:
seems CRS means different things between this function (EPSG code?) and the next function (proj4 code?). Consider unifying?

Reply:
They also have different defaults

Reply:
Not a fan of an integer value that is actually an EPSG code. My preference, when we want to use EPSG codes instead of projection parameters, would be to pass something that also works with the traditional sp/rgdal CRS(). So in this case, the default would be "+init=EPSG:3086" Also, there probably should not be a default crs here. If we are going to use a default, my preference would be EPSG:3857 (web mercator).

Reply:
The `sf` function `st_transform` defines it's `crs` argument that way: 
```
coordinate reference system: integer with the EPSG code, or character with proj4string
```

We can just choose to use the proj4string, and not include a default.

Reply:
Yep. I saw that. "+init=EPSG:3086" is a proj4string version of the same thing.